itou/templates/apply/includes/profile_infos.html

```diff
@@ -1,107 +1,179 @@
+                {% include 'includes/copy_to_clipboard.html' with content=profile.user.phone|cut:" " css_classes="btn-link" matomo_event_attrs=matomo_event_attrs only_icon=True %}
+            {% else %}
+                <i class="text-disabled">Non renseigné</i>
+            {% endif %}
         </li>
-    {% endif %}
-    {% if profile.rqth_employee %}
-        <li class="mb-3">
-            <b>Titulaire de la RQTH</b>
+        <li>
+            <small>Adresse</small>
+            {% if profile.user.address_on_one_line %}
+                <address>{{ profile.user.address_on_one_line }}</address>
+            {% else %}
+                <i class="text-warning">À renseigner</i>
+            {% endif %}
         </li>
-    {% endif %}
-    {% if profile.oeth_employee %}
-        <li class="mb-3">
-            <b>Bénéficiaire de la loi handicap (OETH)</b>
+    </ul>
+
+    <hr>
+    <div class="row mb-3">
+        <div class="col-12 col-sm">
+            <h3 class="mb-0">Situation</h3>
+        </div>
+    </div>
+
+    <ul class="list-data mb-3">
+        <li>
+            <small>Inscrit à France Travail depuis</small>
+            {% if profile.pole_emploi_since %}
+                <strong>{{ profile.get_pole_emploi_since_display|lower }}</strong>
+            {% else %}
+                <i class="text-disabled">Non renseigné</i>
+            {% endif %}
         </li>
-    {% endif %}
-    {% if profile.rsa_allocation_since %}
-        <li class="mb-3">
-            {{ profile.get_has_rsa_allocation_display }} depuis : <b>{{ profile.get_rsa_allocation_since_display|lower }}</b>
+        <li>
+            <small>Identifiant France Travail</small>
+            {% if profile.pole_emploi_id %}
+                <strong>{{ profile.pole_emploi_id }}</strong>
+                {% include 'includes/copy_to_clipboard.html' with content=profile.pole_emploi_id css_classes="btn-link" only_icon=True %}
+            {% else %}
+                <i class="text-disabled">Non renseigné</i>
+            {% endif %}
         </li>
-    {% endif %}
-    {% if profile.ass_allocation_since %}
-        <li class="mb-3">
-            Bénéficiaire de l'ASS depuis : <b>{{ profile.get_ass_allocation_since_display|lower }}</b>
+        <li>
+            <small>Niveau de formation</small>
+            {% if profile.education_level %}
+                <strong>{{ profile.get_education_level_display }}</strong>
+            {% else %}
+                <i class="text-disabled">Non renseigné</i>
+            {% endif %}
         </li>
-    {% endif %}
-    {% if profile.aah_allocation_since %}
-        <li class="mb-3">
-            Bénéficiaire de l'AAH depuis : <b>{{ profile.get_aah_allocation_since_display|lower }}</b>
+        <li>
+            <small class="align-self-start">Autres</small>
+            {% if other_infos %}
+                <strong>
+                    {% for item in other_infos %}
+                        {% if not forloop.first %}<br>{% endif %}
```

Comment:
A priori un seul `<br>` suffit et le `{% if not forloop.last %}<br>{% endif %}` devrait suffire ?